atlast_sc_tests/unit_tests/test_calculator.py

```diff
@@ -143,13 +144,17 @@ def test_update_properties(self, param, new_value,
             # Verify that the derived parameters were updated,
             # where appropriate
             if derived_params_recalculated:
-                calculate_derived_params_spy.assert_called()
-                assert calculator.derived_parameters != \
-                       original_derived_params
+                if finetuned:
+                    calculate_derived_params_spy.assert_called()
+                    assert calculator.derived_parameters == original_derived_params                
+                else:
+                    calculate_derived_params_spy.assert_called()
+                    assert calculator.derived_parameters != \
+                        original_derived_params
```

Comment:
This is for compatibility with the old case of `bandwidth` not triggering an update in the parameters. 
In fact, it won't affect the various parameters checked by `calculate_derived_params_spy` (e.g., Tsys), but only re-calculate the effective `sefd`. Thus, in the `finetune=True` case, we want all the `derived_params` to actually remain the same when changing `bandwidth`, while we want to check for an actual change in all other cases.

A better way to do this would be to recalculate "band-averaged" quantities, of course, but I just went for the quick-n-dirty solution.